packages/web-twig/src/Resources/components/Breadcrumbs/README.md

```diff
@@ -88,19 +88,46 @@ Without lexer:
 ```
 
-## API
+## Breadcrumbs
 
 The Breadcrumbs component works with breadcrumb items passed from parent and renders the content by itself or its
 content can be overridden by any custom block content.
 
-## Breadcrumbs
+### API
 
 | Name          | Type     | Default | Required | Description                                                                                                                                                                                                                                                                       |
 | ------------- | -------- | ------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `elementType` | `string` | `nav`   | ✕        | HTML tag to render                                                                                                                                                                                                                                                                |
 | `goBackTitle` | `string` | —       | ✕        | Title/translation for back link to previous page on mobile. It's essential to be set along with items. If items property is not passed, backlink is to be created within children property. [**Optional DEPRECATED**][Deprecated] Will be **required** in the next major version. |
 | `items`       | `array`  | `[]`    | ✕        | Navigation menu items                                                                                                                                                                                                                                                             |
 
+## BreadcrumbsItem
+
+Use the `BreadcrumbsItem` component for the ordered list as the component's children instead of passing the breadcrumb items array via props:
+
+```twig
+<Breadcrumbs>
+  {% for item in items %}
+    <BreadcrumbsItem isCurrent={{ items.length === index - 1 }} href={{ item.url }}>
+        {{ item.title }}
+    </BreadcrumbsItem>
+  {% endfor %}
+</Breadcrumbs>
+```
+
+### API
+
+| Name               | Type            | Default                         | Required | Description                                           |
+| ------------------ | --------------- | ------------------------------- | -------- | ----------------------------------------------------- |
+| `children`         | `ReactNode`     | —                               | ✕        | Custom content to override items rendering from array |
+| `href`             | `string`        | —                               | ✔        | URL                                                   |
+| `iconEnd`          | `ReactNode`     | `<Icon name="chevron-right" />` | ✕        | Icon element on the end of the item wrapper           |
```

Comment:
there is not iconend only unsafe_iconend. Also reactnode in twig?

Reply:
Fixed.